### DISCUSSION TITLE
Match weather widget main icon to icon pack.

### DISCUSSION THREAD

**Discussion by chip33** [valinet/ExplorerPatcher#1015] [Ideas]
Could the current weather widget webpage icon be swapped to match the new day/night specific taskbar icon?

and maybe even the predicted weather ones too swapped for the MS ones, as the icons are nicer with a colour gradient.
![Screenshot 2022-03-06 175029](https://user-images.githubusercontent.com/48499740/156935483-66e962cd-51a9-493b-884a-542f2ce50fc8.png)
![Screenshot 2022-03-06 175005](https://user-images.githubusercontent.com/48499740/156935487-f43928bc-5bfe-4958-9ff4-98f2b9bdaad9.png)

**Comment by valinet** [comment]
Main icon is already there, kind of, I mean, the code to do it is in there, it's just commented out. I am still debating on this. I mean, yeah, the proper solution would be to swap all of them when the Microsoft theme is active. Probably will implement that, give me some time :D I will take this into consideration into one form or another. Thanks.

**Reply by chip33** [reply]
Thank you for doing this. The new icons look a lot better to me.
They also match nicely with the MS weather widget in the Win10 start menu.

**Reply by valinet** [reply]
Yeah, my biggest complaint with Google’s are that they do not differentiate between day and night for some reason.
